src/cartservice/cartstore/RedisCartStore.cs

```diff
@@ -30,6 +31,7 @@ public class RedisCartStore : ICartStore
         private const string CART_FIELD_NAME = "cart";
         private const int REDIS_RETRY_NUM = 5;
 
+        public ConnectionMultiplexer Redis => redis;
```

Comment:
Might just name this property `ConnectionMultiplexer` just to make things a little more clear